src/db/SQLiteHelper.java

```diff
@@ -41,7 +44,7 @@ public SQLiteHelper() {
     public void createDatabase(String databaseName) {
         this.databaseName = databaseName;
         try {
-            connection = DriverManager.getConnection("jdbc:sqlite:");
+            connection = DriverManager.getConnection("jdbc:sqlite:"+databaseName+".db");
```

Comment:
Omit database name when creating connection will make the `SQLite JDBC` to store the new database in RAM, which allows very fast operations (after that we rewrite the content to disk in `writeDatabaseToDisk()`). 
So I dont think adding a database name here is a good choice.